src/Http/Controllers/CP/Collections/RestoreEntryRevisionController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Statamic\Eloquent\Http\Controllers\CP\Collections;
+
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Statamic\Eloquent\Revisions\RevisionModel;
+use Statamic\Http\Controllers\CP\CpController;
+use Statamic\Revisions\Revision;
+use Statamic\Eloquent\Revisions\WorkingCopy;
+use Statamic\Facades\Revision as Revisions;
+
+class RestoreEntryRevisionController extends CpController
+{
+    public function __invoke(Request $request, $collection, $entry)
+    {
+        $date = Carbon::parse($request->revision);
+        $revisionModel = RevisionModel::where('date', $date)->first();
+
+        if (! $target = $entry->revision($revisionModel->id)) {
+            dd('no such revision', $request->revision);
+            // todo: handle invalid revision reference
+        }
```

Comment:
DD here?

Reply:
Was copied from the cms controller with method override. I guess we can ignore it here.